### ISSUE TITLE
Tracks in dot directories are scanned into the library

### ISSUE THREAD

**Issue by chrisr87** [quodlibet/quodlibet#4254]
Steps to reproduce
------------------
1. Move a song into directory `.Trash`
2. Song will be scanned into library

Expected Output
---------------
Song moved into `.Trash` should not be scanned into the library

Actual Output
-------------
Song moved into `.Trash` was scanned into library

Test System
-----------

### Which version of Quod Libet?
4.5.0

### Which operating system 
Arch Linux

Additional Information
----------------------
I use Syncthing and it automatically moves deleted files to `.Trash`. Yes, I could disable this in Syncthing, but I'd rather not and would prefer that QL not scan folders/files prefixed with `.` like other music players do because it makes more sense to do that or at least, give the option to wildcard exclude folders.

**Comment by dseomn** [comment]
This could be an issue if directories are named after artists/albums, e.g., with [...anabase*](https://musicbrainz.org/artist/1e3564da-a19c-4314-b7a0-5b7d06b8ed12)

**Comment by declension** [comment]
> This could be an issue if directories are named after artists/albums, e.g., with [...anabase*](https://musicbrainz.org/artist/1e3564da-a19c-4314-b7a0-5b7d06b8ed12)

Or [...And Justice For All](https://musicbrainz.org/release/c3ef733e-7dd1-48b3-a60d-1e01df3c6f4e)

**Comment by chrisr87** [comment]
> > This could be an issue if directories are named after artists/albums, e.g., with [...anabase*](https://musicbrainz.org/artist/1e3564da-a19c-4314-b7a0-5b7d06b8ed12)
> 
> Or [...And Justice For All](https://musicbrainz.org/release/c3ef733e-7dd1-48b3-a60d-1e01df3c6f4e)

There are characters from songs that would (I'm assuming) be invalid characters in some file systems as well that need to be stripped or replaced. The artist/album name would still be maintained in the library. But I understand this may cause more problems than it's worth because people would start missing tracks in their library and won't know why.

**Comment by declension** [comment]
Yes. Could probably check for a few well known dodgy ones, or just say that something like `/^\.\w+$/` (i.e. single dot) is to be ignored

**Comment by declension** [comment]
This should have been fixed in #3917
